packages/flutter_tools/templates/app/ios-swift.tmpl/Runner.xcodeproj/project.pbxproj.tmpl

```diff
@@ -325,6 +325,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = {{iosIdentifier}};
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_VERSION = 4.0;
```

Comment:
Is this related?

Reply:
This was missing from the profile build configuration, and broke a swift related test that ran profile